readMe.md

```diff
@@ -0,0 +1,27 @@
+# Pokémon Card Tracker App
+Comprehensive web-app used to track prices & ratings of Pokémon cards. 
+Created by Kobe Yang and Avery Tribbett, guided by Reid Williams.
+---
+## Table of Contents
+* [General Info](#general-info)
+* [Setup](#setup)
+* [File Structure](#file-structure)
+* [Contact](#contact)
+* [Acknowledgments](#acknowledgments)
+---
+## General Info
+---
+## Setup
+---
+## File Structure
+---
+<!-- Not sure if github links are redundant and we can also ask if Reid wants his contact info as well. -->
+## Contact
+Avery Tribbett
+* [LinkedIn](https://www.linkedin.com/in/averytribbett/)
+*[GitHub](https://github.com/averytribbett)
```

Comment:
Add a space after the asterisk here. Markdown is really annoying to work with because it has to have everything correct 

`* [GitHub](https://github.com/averytribbett)`